src/components/header/Header.tsx

```diff
@@ -1,16 +1,18 @@
-import React from 'react'
+import React from "react";
 import './Header.css'
+import logo from '../../assets/meme_depot_logo.png'
+import { NavLink } from 'react-router-dom'
 
-type GreetProps = {
-	name: string
+const Header: React.FC = () => {
+  return (
+    <header>
+      <img src={logo} alt="Meme Depot Logo" className="logo" />
+      <nav>
+        <NavLink exact to="/" className={isActive => "nav-link" + (!isActive? ' unselected': "")} > Home </NavLink>
+        <NavLink exact to="/savedmemes" className={isActive => "nav-link" + (!isActive? ' unselected': "")}> Saved Memes </NavLink>
```

Comment:
Can you please explain whats happening in the <NavLink>? More specifically, the className. Thanks!

Reply:
@mrlobatoman It is toggling the class of the nav links for styling. 
It's the 'fancy' part of NavLinks.  The active tab right now shows orange with an underline while the inactive shows just black text.